Apps/WebClient/test/functional/e2e/cypress/integration/pages/login/KeycloakLogin.js

```diff
@@ -0,0 +1,14 @@
+const { AuthMethod } = require("../../../support/constants")
+
+describe('Keycloak Login', () => {
+    before(() => {
+        cy.login(Cypress.env('keycloak.username'), 
+                 Cypress.env('keycloak.password'), 
+                 AuthMethod.KeyCloak)
+        cy.get('#covid-modal___BV_modal_header_ > .close').click()
```

Comment:
this will fail if the covid modal doesn't display

Reply:
removed covid modal